gallery/src/components/NxTransferListHalf/NxTransferListHalfPage.tsx

```diff
@@ -276,6 +278,13 @@ const NxTransferListPage = () =>
                         liveExample={NxTransferListHalfOrderingWithDisableTransferExample}>
       Demonstrates an ordering <NxCode>NxTransferListHalf</NxCode> with transfer feature disabled.
     </GalleryExampleTile>
+    <GalleryExampleTile title="Long List Example"
+                        id="nx-transfer-list-half-long-list-example"
+                        codeExamples={nxTransferListHalfLongListExample}
+                        liveExample={NxTransferListHalfLongListExample}>
+      Demonstrates an <NxCode>NxTransferListHalf</NxCode> with a long list. Note that even using a long list,
+      rendering and filtering are not affected.
```

Comment:
Maybe should reword slightly to make it clear that you mean affected in a performance sense

Reply:
> Maybe should reword slightly to make it clear that you mean affected in a performance sense

done at 79dd73fc26f2dd718a4556536c44587e7d0e0825